src/main/kotlin/no/nav/familie/ba/mottak/hendelser/LeesahConsumer.kt

```diff
@@ -45,37 +60,57 @@ class LeesahConsumer(val taskRepository: TaskRepository) {
                                 cr.topic(), cr.partition(), cr.offset(), cr.value().hentOpplysningstype(), cr.value().hentAktørId(), cr.value().hentEndringstype())
                     }
                 }
-            } else if (cr.value().erFødsel())
+                hendelsesloggRepository.save(Hendelseslogg(cr.offset(),cr.value().hentHendelseId(),cr.value().hentAktørId(),cr.value().hentOpplysningstype(),cr.value().hentEndringstype()))
```

Comment:
Er det behov for all denne infologgene når man lagrer de i databasen?

Reply:
@hensol